### PR TITLE
added teru_introduction

### PR DESCRIPTION
**Pull request by teru-1f8**

### CODE REVIEW

File: content/blog/teru-self-introduction-teru/getdate.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (1f8) [inline content/blog/teru-self-introduction-teru/getdate.html:1]: This file isn't part of anything. Should be removed.

**Review by 1f8 — CHANGES_REQUESTED** [review]

File: content/blog/teru-self-introduction-teru/teru_introduction.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+title: Self-introduction : Teru
```
Comment (1f8) [inline content/blog/teru-self-introduction-teru/teru_introduction.md:1]: This is missing `---` before and after the metadata header. See other entries for examples.

Comment (1f8) [inline content/blog/teru-self-introduction-teru/teru_introduction.md:1]: The `:` sign is a marker in the header. If you have `:` in the title, you need to surround it by quotes. Should be:
`title: "Self-introduction: Teru"`

**Review by 1f8 — CHANGES_REQUESTED** [review]
Please make some changes.

**Comment by teru-1f8** [comment]
I changed it and pushed.

**Review by 1f8 — APPROVED** [review]

**Pull request merged**